canonical-web-and-design/vanilla-framework
# Add application icons

## Done

Added a subset of application icons per [this request from the JAAS team](https://github.com/canonical-web-and-design/jaas-dashboard/issues/453#issuecomment-630949936), that are not included in Vanilla by default. 

Fixes #3426 

## QA

- Pull code
- Run `./run`
- Open https://vanilla-framework-3436.demos.haus/docs/patterns/icons#application
- See the new icons are present, and that there is documentation on how to import them is present at https://vanilla-framework-3436.demos.haus/docs/patterns/icons#import
- See that the examples also look correct: https://vanilla-framework-3436.demos.haus/docs/examples/patterns/icons/icons-application and https://vanilla-framework-3436.demos.haus/docs/examples/standalone/patterns/icons/icons-application
